spec/models/adoption_app_spec.rb

```diff
@@ -56,4 +57,36 @@
       expect(create(:adoption_app)).to be_valid
     end
   end
+
+  describe '.adopter_age' do
+    it 'returns nil when birth_date is nil' do
+      adoption_app.birth_date = nil
+
+      expect(adoption_app.adopter_age).to be nil
+    end
+
+    it 'returns 10 years when birth_date is 10 years before current date' do
+      adoption_app.birth_date = 10.years.ago.to_date
+
+      expect(adoption_app.adopter_age).to eq("10 years")
+    end
+
+    it 'returns 9 years when birth_date is 1 month before 10 years from current date' do
+      adoption_app.birth_date = (10.years.ago + 1.month).to_date
+
+      expect(adoption_app.adopter_age).to eq("9 years")
+    end
+
+    it 'returns 9 years when birth_date is 1 day before 10 years from current date' do
```

Comment:
@nickcianci turns out this test will fail on late night CI runs :)    If UTC date and ET date are different this test will fail.  Given the org is fully Eastern Time based, it's probably best to honor local timezone of the app for determining age.  

I've added a quick time helper to consistently break the test which should make debugging a bit easier.  You should be able to push / pull directly from this feature branch.

Reply:
I'm not able to replicate. Isn't it already honoring local timezone? I'm not honoring local timezone by using `10.years.ago + 1.day`?

Reply:
Not sure, are you able to see https://travis-ci.org/ophrescue/RescueRails/jobs/647725489#L1778  ?

Reply:
yes, can we print out the current datetime from the build server when running the test?

Reply:
Sure can, just pushed some `puts`

Reply:
Oh, I think I'm supposed to used Date.current instead of Date.today maybe.

Reply:
https://stackoverflow.com/questions/12748626/when-should-datetime-now-utc-vs-time-current-utc-be-used-in-rails

Reply:
Yeah I think that's it
```
Date.current is 2020-01-29
Date.today is 2020-01-30
```

Reply:
Fix pushed

Reply:
Timecop is great for this stuff too